ironfish/src/migrations/data/027-account-created-at-block/new/headValue.ts

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import { IDatabaseEncoding } from '../../../../storage'
+
+export type HeadValue = {
+  hash: Buffer
+  sequence: number
+}
+
+export class NullableHeadValueEncoding implements IDatabaseEncoding<HeadValue | null> {
+  readonly nonNullSize = 32 + 4
```

Comment:
can and should `nonNullSize` be derived from other constants or anything? I think the answer is "no, literal `32 + 4` is what makes the most sense here" but worth a double-check

Reply:
I looked around for any constants we have for the length of hashes (32 bytes), but didn't find any 🤔 I suspect that if we add one we'll want to replace a lot of `32` literals

For the `4` I think the literal `4` might make the most sense since it's just referring to the byte length of the 32-bit integer